### ISSUE TITLE
require higher sass version

### ISSUE THREAD

**Issue by darix** [octopress/genesis-theme#27]
3.2.19 wasnt enough to render the files part of octopress/this theme. sass 3.4.13 on the other hand worked.

**Comment by lukaszklis** [comment]
It's a dependency inherited from https://github.com/jekyll/jekyll-sass-converter/blob/master/jekyll-sass-converter.gemspec. After cloning the repository and running `bundle`, it installed sass in 3.4.13:
```
➜  genesis-theme git:(master) bundle
Fetching gem metadata from https://rubygems.org/..........
Resolving dependencies...
...
Installing sass 3.4.13
```

**Comment by darix** [comment]
```gem install --conservative``` and not everyone is using bundler either. our rubygem rpms are honoring dependencies. and in this case 3.2 was enough but it is actually a non working version. as such the dependency should be bumped. also while jekyll-sass-convert is correct with ```~> 3.2``` because i could write 3.2 compatible code. the code in this theme isnt. as such it should require a newer sass version.

**Comment by imathis** [comment]
I just released version 0.0.9 which addresses this issue and bumps the Sass dependency up to 3.4. Thanks!

**Comment by darix** [comment]
On 2015-04-07 13:25:46 -0700, Brandon Mathis wrote:
> I just released version 0.0.9 which addresses this issue and bumps the Sass dependency up to 3.4. Thanks!

Thank you. Updating my rpm.

   darix

-- 
           openSUSE - SUSE Linux is my linux
               openSUSE is good for you
                   www.opensuse.org
